swap_meet/vendor.py

```diff
@@ -1,2 +1,210 @@
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None) :
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        if not item:
+            return item
```

Comment:
Thank you for this. I'm used to do this all the time since I've have mostly worked with hard typed languages like Java or C#. I would follow your advice and return the right type or None next time.